packages/providers/fal/text-to-video/fal_text_to_video/models/base.py

```
@@ -0,0 +1,214 @@
+                "video_url": video_url,
+                "local_path": str(local_path),
+                "filename": filename,
+                "prompt": prompt,
+                "model": self.model_key,
+                "model_name": self.display_name,
+                "cost_usd": cost,
+                "parameters": validated_params,
+                "metadata": result
+            }
+
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)